doc/source/credits.rst

```diff
@@ -79,13 +79,16 @@ Vyas Ramasubramani - **Lead developer**
 * Converted all docstrings to Google format, fixed various incorrect docs.
 * Debugged and added rotational autocorrelation code.
 * Added MSD module.
+* Wrote NeighborQuery, _QueryArgs, NeighborQueryResult classes.
+* Wrote PairCompute and SpatialHistogram parent classes.
+* Wrote ManagedArray classes.
```

Comment:
ManagedArray class, singular. Also C++ histogram related classes and the neighbor iterator infrastructure.